modules/ROOT/pages/environments.adoc

```diff
@@ -54,10 +62,40 @@ To view the client ID and secret for your environment:
 
 . Log in to the Anypoint Platform as an administrator.
 . Go to Access Management.
-. From the left-hand menu, click *Environments*.
+. In the menu on the left, click *Environments*.
 . Click the name of the environment whose client ID/secret you want to view.
 
 [NOTE]
 --
 To change the client ID or client secret of an environment, you must contact your customer support representative.
 --
+
+[[add-an-idp-to-an-existing-environment]]
+== Associate a Client Provider with an Existing Environment
+
+If your organization administrator provides multiple xref:managing-api-clients.adoc[client providers] for your organization, you can add them to existing environments.
+
+To associate a client provider with an existing environment:
+
+. Sign in to Anypoint Platform using an account that has the organization administrator role.
+. Go to Access Management.
+. In the menu on the left, click *Environments*.
+. Click the name of the environment.
+. In the *Client Provider* field, select at least one client provider.
+. If you added more than one client provider, click *Set as default* to designate a default client provider.
+. Click *Update*.
+
+You can now configure client providers for APIs. 
+
+[[remove-an-idp-from-an-environment]]
+== Remove a Client Provider from an Environment
+
+You can remove a client provider from an environment at any time.
```

Comment:
Should we provide examples of the circumstances under which you might want to remove?

Reply:
Dev says there are lots of reasons to delete; I think many of them would probably be too niche to document. From our side, there isn't a reason to do so, but they could want to delete to change client providers or something.